src/examples/table/TableScroll.tsx

```diff
@@ -32,30 +36,25 @@ const rowData: IRow[] = Array.from(Array(100)).map((row, index) => ({
 }));
 
 const Example = () => (
-  <div style={{ overflowX: 'auto' }}>
-    <Table style={{ minWidth: 500 }}>
-      <Head>
+  <div style={{ maxHeight: 500, overflowY: 'auto' }}>
+    <Table>
```

Comment:
Why was the `minWidth` removed?

Reply:
No particular reason than when an extra `div` was removed.